client/src/context/store.js

```diff
@@ -26,20 +32,23 @@ const StateProvider = ({ children }) => {
           isAuthenticated: true,
           loading: false,
           user: payload,
+          socket
```

Comment:
beacuse you return ...state, do you need to keep passing it through?

Reply:
I thought so too but for some reason state.socket was returning undefined in Lobby.js unless I explicitly added it in like this.